cmd/server/assets/admin/users/index.html

```diff
@@ -44,7 +44,14 @@
             <tbody>
             {{range $users}}
               <tr>
-                <td>{{.Name}}</td>
+                <td>
+                  {{.Name}}
+                  {{if .Admin}}
+                  <span class="ml-1 badge badge-pill badge-primary">System admin</span>
+                  {{else if .IsRealmAdmin}}
```

Comment:
I don't think this is if/else, a single user could be both

Reply:
Idk if that's useful. A System Admin is likely both (and can join any realm as admin), so it seems a little redundant to show both.

Reply:
I think it's useful to show which realms a person is admin and member of

Reply:
We will, but in this list it would be hard to show all the realm names etc.
This will become a link to a detail page (that looks more similar to the My Account page with a list of realms)

Reply:
my vote is the same as seth - show both pills in that case

Reply:
well that's 2-1 then. Both pills shown.